config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```
@@ -316,7 +316,7 @@ presubmits:
         - name: RUNTIME_CONFIG
           value: '{"api/alpha":"true", "api/ga":"true"}'
         - name: LABEL_FILTER
-          value: "!Slow && !Disruptive && !Flaky && Feature: isSubsetOf Alpha"
+          value: "Feature: isSubsetOf OffByDefault && !BetaOffByDefault && !Slow && !Disruptive && !Flaky"
```

technically this would be the direct equivilant:
```suggestion
          value: "Feature: isSubsetOf OffByDefault && !Beta && !Slow && !Disruptive && !Flaky"
```

---

I think:
> "Runs tests with no special requirements other than alpha feature gates in a KinD cluster where alpha feature gates and APIs are enabled."

matches the current change though

---

there is a oneline PAtrick has in the e2e doc that gives you the output to compare

---

I'm saying this is *intentionally* not identical, because we can more-correctly match the intent now.

The question is do we want that, or match the previous slightly "wrong" state (which excluded any test depending on an on-by-default-beta despite not disabling on-by-default beta, and not being described that way in the description)

(the quote above is the current job description)

---

> there is a oneline PAtrick has in the e2e doc that gives you the output to compare

I still haven't found this, and the alternative I hacked up locally isn't nice for diffing (this command is in the featuregate labeling proposal doc at the end). 

Ideally we'd have a way to list matching test names *considering filters* in a trivially diffable way, but `list-tests` doesn't seem to do that.

---

`Feature: isSubsetOf OffByDefault && !Beta ` and `Feature: isSubsetOf OffByDefault && !BetaOffByDefault` select the same number tests currently. 

There are only 3 tests that are `[Beta]` and `[Alpha`, and those all have `[Feature:DRAAdminAccess]` so they're excluded from these jobs.